### ISSUE TITLE
Improve README on pre-existing data

### ISSUE THREAD

**Issue by debackerl** [minio/minio#9906]
**Is your feature request related to a problem? Please describe.**
I would like to run Minio in front of a directory which is updated by another process than Minio.

**Describe the solution you'd like**
Current documentation about "pre-existing data" states that it's fine to start Minio with a directory containing files already (https://github.com/minio/minio#pre-existing-data). However, it does not state if new files can be added, or if files can be deleted by other processes than Minio.

**Describe alternatives you've considered**
N/A

**Additional context**
Basically, the goal is to use Minio as an integration between old file-based batched, and microservices running in containers. Those micros-services would connect to Minio to list and read files provided by old batches.

Thank you!

**Comment by harshavardhana** [comment]
> I would like to run Minio in front of a directory which is updated by another process than Minio.

This is not allowed @debackerl you cannot edit the backend without going through MinIO - if you do then results are undefined.

**Comment by kannappanr** [comment]
@debackerl closing this issue based on the comment above. Please feel free to reach out to us if you have any questions.
